pkg/operator/ceph/object/objectstore.go

```diff
@@ -214,6 +214,11 @@ func getMultisiteForObjectStore(clusterdContext *clusterd.Context, spec *cephv1.
 		return realm.Name, zonegroup.Name, zone.Name, nil
 	}
 
+	// TODO: check or fix https://github.com/rook/rook/issues/9750
```

Comment:
Move this at the beginning of the function so we don't call anything else and return quickly?

Reply:
Sure, I will move this to beginning of the function since multisite is not applicable for external cluster atm